package/MDAnalysis/transformations/rotate.py

```diff
@@ -144,38 +148,47 @@ def __init__(self,
         self.angle = np.deg2rad(self.angle)
         try:
             self.direction = np.asarray(self.direction, np.float32)
-            if self.direction.shape != (3, ) and \
-               self.direction.shape != (1, 3):
-                raise ValueError('{} is not a valid direction'
-                                 .format(self.direction))
-            self.direction = self.direction.reshape(3, )
+            if self.direction.shape != (3,) and self.direction.shape != (1, 3):
+                raise ValueError(
+                    "{} is not a valid direction".format(self.direction)
+                )
+            self.direction = self.direction.reshape(
```

Comment:
I'm not going to block on this.. but surely this is a single line thing?

Reply:
Nope. With the indentation this is 86 characters long. In https://github.com/MDAnalysis/mdanalysis/pull/4717 we decided to stick with 79 characters, hence the formatting. I'd be happy to reconsider the line length, but I think that is something to do once the per-module formatting is over (if there is consensus).